RNTester/Podfile

```diff
@@ -35,11 +35,16 @@ end
 
 source 'https://cdn.cocoapods.org/'
 
-platform :ios, '9.0'
-
 require_relative '../scripts/autolink-ios'
```

Comment:
Still seems wrong to pull in and use "autolink-ios" for both Mac and iOS, but given that we're trying to reduce diffs between us and facebook, it's probably not a blocking issue to get cleaned up right now. Ideally we'd have an autolink-apple, autolink-mac, and autolink-ios

Reply:
> Still seems wrong to pull in and use "autolink-ios" for both Mac and iOS, but given that we're trying to reduce diffs between us and facebook, it's probably not a blocking issue to get cleaned up right now. Ideally we'd have an autolink-apple, autolink-mac, and autolink-ios

Just in case this was missed before, this comment addresses the plan https://github.com/microsoft/react-native/pull/211#discussion_r363768019